src/modules/video_player/index.js

```diff
@@ -3,23 +3,56 @@ const watcher = require('../../watcher');
 const settings = require('../../settings');
 const keyCodes = require('../../utils/keycodes');
 
+function stepPlaybackSpeed($player, faster) {
```

Comment:
instead of this really tedious dom manipulation.. you can just get the current player from our twitch util

```
const currentPlayer = twitch.getCurrentPlayer();
if (!currentPlayer) return;
currentPlayer.player.setPlaybackRate(2);
```